tests/test_web/test_webapi.py

```diff
@@ -2,26 +2,40 @@
 
 import pytest
 import responses
+import numpy as np
 from _pytest import monkeypatch
+
 import tidy3d as td
 from responses import matchers
 from tidy3d import Simulation
-from tidy3d.exceptions import SetupError
-from tidy3d.web.environment import Env
-from tidy3d.web.webapi import delete, delete_old, download, download_json, run, abort
-from tidy3d.web.webapi import download_log, estimate_cost, get_info, get_run_info, get_tasks
-from tidy3d.web.webapi import load, load_simulation, start, upload, monitor, real_cost
-from tidy3d.web.container import Job, Batch
-from tidy3d.web.asynchronous import run_async
+from tidy3d.web.api.tidy3d_stub import Tidy3dStub
+from tidy3d.web.core.environment import Env
+from tidy3d.web.api.webapi import delete, delete_old, download, download_json, run, abort
+from tidy3d.web.api.webapi import download_log, estimate_cost, get_info, get_run_info, get_tasks
+from tidy3d.web.api.webapi import load, load_simulation, start, upload, monitor, real_cost
+from tidy3d.web.api.container import Job, Batch
+from tidy3d.web.api.asynchronous import run_async
 
 from tidy3d.__main__ import main
+from tidy3d.web.core.types import TaskType
+from tidy3d.components.source import PointDipole, GaussianPulse
+from tidy3d.components.grid.grid_spec import GridSpec
+from tidy3d.components.data.sim_data import SimulationData
+from tidy3d.components.data.monitor_data import FieldData
+from tidy3d.components.data.data_array import ScalarFieldDataArray
+from tidy3d.components.monitor import FieldMonitor
 
 TASK_NAME = "task_name_test"
 TASK_ID = "1234"
+FOLDER_ID = "1234"
 CREATED_AT = "2022-01-01T00:00:00.000Z"
 PROJECT_NAME = "default"
 FLEX_UNIT = 1.0
 EST_FLEX_UNIT = 11.11
+FILE_SIZE_GB = 4.0
+
+path = "tidy3d.web.core.task_core"
```

Comment:
Maybe rename to something like `task_core_path`?

Reply:
Modified.